### DISCUSSION TITLE
Skip 1.0 released!

### DISCUSSION THREAD

**Discussion by marcprux** [skiptools/skip#194] [Announcements]
Skip 1.0 has been released! See the announcement at: https://skip.tools/blog/skip-1_0-release/

For those of you who have been with us through the early access days, any major version number bump requires updating your `Package.swift` dependencies. You can see the full instructions for Skip 0.x projects at: https://skip.tools/docs/update-guide. 

Our own frameworks and sample apps have been updated to use Skip 1.0.

Also, with this release, we have introduced a new Indie tier for solo developers. You can see all the details at https://skip.tools/pricing/. And for paid license holder, we will be be sending out invitations to a new dedicated Slack support channel.

As always, Happy Skipping! 🥳

**Comment by marchy** [comment]
Wow I so praise you guys for taking this fantastic transpiler-based approach!

Been using Swift-to-Kotlin porting (through a basic transpiler approach – separate from native toolsets) to achieve a too-rare-in-the-industry dual-platform eng process that has worked wonders in the past, but I could not be more stoked to see a fully-stacked solution get wrapped around it from toolset to framework-equivalency (ie: Combine).

Can't wait to start trying this out in action!
